taskiq/receiver/receiver.py

```diff
@@ -294,7 +313,7 @@ async def run_task(  # noqa: C901, PLR0912, PLR0915
         # Assemble result.
         result: "TaskiqResult[Any]" = TaskiqResult(
             is_err=found_exception is not None,
-            log=None,
+            log=log,
```

Comment:
The log here seems to be just the normal log in the task function, do we need to collect the exceptions generated by `NoResultError` and `BaseException` as well?